Readme.md

```diff
@@ -40,23 +41,105 @@ according to your preferences.
 * [Object.freeze, Object.preventExtensions, Object.seal, with, eval](#objectfreeze-objectpreventextensions-objectseal-with-eval)
 * [Getters and setters](#getters-and-setters)
 
-## 2 Spaces for indention
 
-Use 2 spaces for indenting your code and swear an oath to never mix tabs and
-spaces - a special kind of hell is awaiting you otherwise.
+## Code Formatting
 
-## Newlines
+In general, we will follow [Google's code formatting guide][googleformatting].
+
+[googleformatting]: https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting
+
+### 80 characters per line
+
+Limit your lines to 80 characters wide. Except the following exceptions:
+
+- URL in comments;
+
+- Regular expression created using `/` notation; *(Such occurrences should be
+  declared as constants near the beginning of the file.)*
+
+-  Script dependencies, such as `require()`; *(Such occurrences should be
+  declared near the beginning of the file.)*
+
+### Indentions
+
+Use **two** spaces for indenting your code; wrapped lines should be indented  
+either left-aligned to a sibling expression above, or **four** spaces (not
+**two** spaces) deeper than a parent expression (where "sibling" and "parent"
+refer to parenthesis nesting level).
```

Comment:
I mentioned this in a comment elsewhere as well, but I agree with @shezarkhani. Unless this can be done automatically, I'm afraid it will become very difficult to manage with all of the rules around spaces.